com.enjoyingfoss.Feeel.yml

```diff
@@ -0,0 +1,47 @@
+app-id: com.enjoyingfoss.Feeel
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: feeel
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --own-name=com.enjoyingfoss.feeel
```

Comment:
You should just use that d-bus name as the app-id as it is granted byb default. (yes it's case sensitive)

Reply:
@hfiguiere OK, should be fixed.